pontos/updateheader/updateheader.py

```diff
@@ -343,25 +354,27 @@ def main(args: Optional[Sequence[str]] = None) -> None:
         sys.exit(1)
 
     for file in files:
-        if changed:
-            try:
-                year = _get_modified_year(file)
-            except PontosError:
-                term.warning(
-                    f"{file}: Could not get date of last modification"
-                    f" via git, using {year} instead."
-                )
 
         try:
             if file.absolute() in exclude_list:
                 term.warning(f"{file}: Ignoring file from exclusion list.")
             else:
+                if changed:
+                    try:
+                        year = _get_modified_year(file)
+                    except PontosError:
+                        term.warning(
+                            f"{file}: Could not get date of last modification"
+                            f" via git, using {year} instead."
+                        )
```

Comment:
```suggestion
                        year = _get_modified_year(file)
                    except PontosError:
                        term.warning(
                            f"{file}: Could not get date of last modification"
                            f" via git, using {year} instead."
                        )
                        parsed_args.year
```

Independent from your PR: I wonder if we need to set it like this, because the function is overwriting the argument `year` if `_get_modified_year(file)` is executed?

Reply:
I looked into it and it doesn't seem to change behaviour in any way (looks like if _get_modified_year throws exception then the year is not being set by it and it stays default).